oci/spec_opts_windows.go

```diff
@@ -100,3 +100,32 @@ func escapeAndCombineArgs(args []string) string {
 	}
 	return strings.Join(escaped, " ")
 }
+
+// WithWindowsDevice adds a device exposed to a Windows (WCOW or LCOW) Container
+func WithWindowsDevice(IDType, ID string) SpecOpts {
+	return func(_ context.Context, _ Client, _ *containers.Container, s *Spec) error {
+		if len(IDType) == 0 {
+			return errors.New("missing IDType")
+		}
+		if s.Windows == nil {
+			s.Windows = &specs.Windows{}
+		}
+		s.Windows.Devices = append(s.Windows.Devices, specs.WindowsDevice{IDType: IDType, ID: ID})
+		return nil
+	}
+}
+
+// WithLCOWUVMDevice adds a device exposed from the LCOW kernel to an LCOW Container
+func WithLCOWUVMDevice(path string) SpecOpts {
```

Comment:
That's fair. I'm happy to actually totally drop the LCOW stuff here, since it's not exposed outside the `oci` API. It was mainly that I _already_ had it, and I figured if someone wanted to actually implement an alternative CRI setup, e.g. up-porting https://github.com/jterry75/cri/tree/windows_port to work with current containerd (if it doesn't already), then at least the non-CRI stuff would be in-place.

Callers can always manually add things to Linux.Devices (or implement their own `WithLinuxDevice`, really) if they want to trigger that code in OpenGCS, once they have called `oci.GenerateSpecWithPlatform(..., "linux")`.